dcpy/lifecycle/package/validate.py

```diff
@@ -282,15 +286,6 @@ def validate_package(
                 )
             )
 
-    unique_row_counts = {v.stats.row_count for v in validations}
```

Comment:
What's the story here?

Reply:
see [commit message](https://github.com/NYCPlanning/data-engineering/pull/890/commits/c820a4992f674fec6eebdcb7fb9974b30dbd66e1)

Reply:
it feels cruel to fully delete it, but maybe this is a check we'd rather save for our upcoming `pandera`ish approach to checks like this?

Reply:
I do think that this probably a check worth saving, but with an option to ignore differing row counts. Perhaps we could set a flag in the metadata at the package level about whether row counts should be the same. However, the pandera approach isn't going to help at all with this, as it's a check that goes across dataset_files. 

So I'd leave this line in place, but not throw an exception during validation if there are differing row counts.